### DISCUSSION TITLE
Table 'nc_roles' already exists error [May 19]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2168] [Discord Threads]
> 🙋‍♂️ **Alsu**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### from what version are you migrating and what is the your NC_DB variable ?  
#  
> 🙋‍♂️ **Alsu**  
#### I am going from v 0.84.16 to 0.90.11. I am just passing my database name variable.  
#  
> 🙋‍♂️ **Alsu**  
#### on my docker file I put latest, I did not specify numbers, and it went to    
#### PackageVersion: **0.84.15**  
#  
> 💁 **o1lab**  
#### may be you have to remove the image from your machine  
#  
> 💁 **o1lab**  
#### it should have picked the latest otherwise  
#  
> 🙋‍♂️ **Alsu**  
#### I am building it with jenkins, and I am cleaning before building it. When I specify which version to update to, it gives me correct one  
#  
> 🙋‍♂️ **Alsu**  
#### removing all system tables (nc_*) worked, I was able to pull the latest version  
#  
> 💁 **o1lab**  
#### (deleted it.. since its not advisable to show the NC_DB 🙂 - I understand some credentialas was redacted .. but yet)  
#  
> 💁 **o1lab**  
#### > removing all system tables (nc_*) worked, I was able to pull the latest version   
#### great  
#  
> 🙋‍♂️ **Alsu**  
#### It is still not ideal, because users and projects are gone, but workaround, which requires to insert that data into the new tables manually  
#  
> 💁 **o1lab**  
#### Did you happen to manually fiddle with NC_DB database (like removing or adding tables or data within) when error happened  
#  
> 💁 **o1lab**  
#### it should automatically role up to the latest version  
#  
> 💁 **o1lab**  
#### the err mentioned was the first time I have seen  
#  
> 🙋‍♂️ **Alsu**  
#### to recap, I faced several issues: 1. pulling latest -- pulled v 0.84.15, when I was already in 0.84.16. 2. first time trying to update gave me issue with nc_projects table -- i deleted it, tried again, gave me error in nc_roles. I deleted all system tables, and pull the latest, it worked, but updated to 0.84.15 version, but specifying which version to update to, creates tables with _v2 and no more issues.  
#  
> 🙋‍♂️ **Alsu**  
#### thank you for looking into my issues with me!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/976533626793623613)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
